### DISCUSSION TITLE
p/4905349aa1c646c3a2bf4b013651eec4/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#50] [Announcements]
# p/4905349aa1c646c3a2bf4b013651eec4/

上周开始了全国旅行的第一周，接下来将前往江浙沪皖地区，其中最值得期待的是安徽的黄山。  上海由于赶上清明节最后一天，没能买票爬上泰山，所以 6 号早上改变计划直接前往了上海。第一次来到上海，被高楼大厦与城市绿化的协调震撼到了，到处都是花花草草，而且看起来很规整。之前在北京广州这种大城市生活，却没有体会到过这种感觉。 随手拍的公路，可以看到摩天大楼、道路、高架桥等现代城市的元素和两侧的树木很协调。

https://blog.rxliuli.com/p/4905349aa1c646c3a2bf4b013651eec4/

**Comment by Jaaayden** [comment]
可以申请一下友链吗？
链接：https://xxu.do
站点标题: Jayden’s site
站点描述: 为天地立心,为生民立命, 为往圣继绝学,为万世开太平。
头像: https://github.com/Jaaayden.png
名字: Jayden

**Reply by rxliuli** [reply]
ok，稍后吾辈添加一下。

**Reply by rxliuli** [reply]
已添加

<img width="946" alt="image" src="https://github.com/rxliuli/blog/assets/24560368/40dc5fc6-1815-4894-90ef-74469e524f67">
